salesapp.js

```diff
@@ -1,16 +1,38 @@
 'use strict';
 
-function CookieStore(idTag, storeName, minHourCustomers, maxHourCustomers, avgCookiesPerCustomer){
-  this.idTag = idTag;
+//-------------------------CONSTRUCTOR OBJECT-------------------
+function CookieStore(storeName, minHourCustomers, maxHourCustomers, avgCookiesPerCustomer){
   this.storeName = storeName;
   this.minHourCustomers = minHourCustomers;
   this.maxHourCustomers = maxHourCustomers;
   this.avgCookiesPerCustomer = avgCookiesPerCustomer;
   this.hourlyCookies = [];
   this.totalSales = 0;
 };
+//-------------------------------FORM----------------------------
+var storeFormEl = document.getElementById('new-store-form');
```

Comment:
I would maybe group global, fixed-value variables like this up at the very top so that they're easy to find.